Description:
 - Updated imports of BeautifulSoup and ClientForm to point to the main
   Python modules.
Author: Ignace Mouzannar <mouzannar@gmail.com>
Last-Update: 2009-12-14

--- a/scrapy/http/response/text.py
+++ b/scrapy/http/response/text.py
@@ -7,7 +7,7 @@
 
 import re
 
-from scrapy.xlib.BeautifulSoup import UnicodeDammit
+from BeautifulSoup import UnicodeDammit
 
 from scrapy.http.response import Response
 from scrapy.utils.python import memoizemethod_noargs
--- a/scrapy/tests/test_utils_response.py
+++ b/scrapy/tests/test_utils_response.py
@@ -1,6 +1,6 @@
 import unittest
 
-from scrapy.xlib.BeautifulSoup import BeautifulSoup
+from BeautifulSoup import BeautifulSoup
 from scrapy.http import Response, TextResponse
 from scrapy.utils.response import body_or_str, get_base_url, get_meta_refresh, \
     response_httprepr, get_cached_beautifulsoup
--- a/scrapy/xlib/ClientForm.py
+++ b/scrapy/xlib/ClientForm.py
@@ -919,7 +919,7 @@
 try:
     if sys.version_info[:2] < (2, 2):
         raise ImportError  # BeautifulSoup uses generators
-    from scrapy.xlib import BeautifulSoup
+    import BeautifulSoup
 except ImportError:
     pass
 else:
--- a/scrapy/http/request/form.py
+++ b/scrapy/http/request/form.py
@@ -8,7 +8,7 @@
 import urllib
 from cStringIO import StringIO
 
-from scrapy.xlib.ClientForm import ParseFile
+from ClientForm import ParseFile
 
 from scrapy.http.request import Request
 from scrapy.utils.python import unicode_to_str
--- a/scrapy/utils/response.py
+++ b/scrapy/utils/response.py
@@ -15,7 +15,7 @@
 
 from scrapy.utils.markup import remove_entities
 from scrapy.utils.url import safe_url_string, urljoin_rfc
-from scrapy.xlib.BeautifulSoup import BeautifulSoup
+from BeautifulSoup import BeautifulSoup
 from scrapy.http import Response, HtmlResponse
 
 def body_or_str(obj, unicode=True):
